plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -40,7 +47,6 @@ public IcebergNessieCatalogConfig setDefaultReferenceName(String defaultReferenc
         return this;
     }
 
-    @NotNull
```

Comment:
Is the removal of the annotation intentional?
Is it related to the changes from this commit?

Reply:
there was a compiler warning at the caller of the `catalogConfig.getServerUri()` that we are checking for the null for an API with @NotNull annotation. After analyzing the code, when user forget configure, We don't have a default value for the uri. Hence, it can be null. Hence, I removed this annotation.

Reply:
reverted.